dm/dm/worker/subtask.go

```diff
@@ -268,6 +268,8 @@ func (st *SubTask) StartValidator(expect pb.Stage) {
 	}
 	st.Lock()
 	defer st.Unlock()
+	// refetch the validation config
```

Comment:
what's this comments for? subtask config is fetched in here
https://github.com/pingcap/tiflow/blob/0a4fc06efc74bc5afdc81d20e6bc872946b597ef/dm/dm/worker/source_worker.go#L1276-L1277

might need to update cfg of validator to update validation mode, we can do this later.

Reply:
Merging mistake😂 I'll fix it